pkg/templates/kubechainsaw/template_test.go

```
@@ -0,0 +1,207 @@
+
+	p := params.Params{}
+	checkFn, err := analyze(p)
+	require.NoError(t, err)
+
+	checkFn(ctx, ctx.Objects()[0])
+
+	diags := checkFn(ctx, ctx.Objects()[1])
+	assert.Empty(t, diags, "role-b should have no findings")
+}
+
+
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)